### DISCUSSION TITLE
Ανακοινώσεις 2ης, 3ης εβδομάδας

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1303] [Q&A] [answered]
Η τελευταία ανακοίνωση για τις ομάδες είναι από προηγούμενο έτος. Δεν ξέρω αν έγινε κατάλάθος αλλά είπα να σας το πω.
Επίσης παρόλο που ξέρουμε τι πρέπει να κάνουμε αναρωτιόμουν γιατί δεν βγήκε ανακοίνωση για τα παραδοτέα 2ης και 3ης εβδομάδας.

**Comment by epidrome** [comment, marked as answer]
#1305 

Καθώς είναι αργία σήμερα, ο σχολιασμός των παραδοτέων θα γίνει μαζεμένος για τις εβδομάδες 2,3 την επόμενη Δευτ-Τρι. #1164 

Επίσης, δεν χρειάζεται να περιμένετε τις εβδομαδιαίες ανακοινώσεις, αφού το αναλυτικό πρόγραμμα είναι ήδη αναρτημένο
